doc/user_guide/user_guide.md

```diff
@@ -377,6 +377,40 @@ The minimum requirements for a regular user in order to be able to access the RL
 * User is allowed to create sessions (`GRANT CREATE SESSION`)
 * User can execute `SELECT` statements on the Virtual Schema (`GRANT SELECT`)
 
+### Adapter Capabilities
+
+RLS is based on Exasol's Virtual Schema. Which constructs are pushed-down is decided by the optimizer based on the original query and on the capabilities reported by the Virtual Schema adapter (i.e. the software driving RLS).
+
+RLS defines the following capabilities:
+
+* `SELECTLIST_PROJECTION`
+* `AGGREGATE_SINGLE_GROUP`
+* `AGGREGATE_GROUP_BY_COLUMN`
+* `AGGREGATE_GROUP_BY_TUPLE`
+* `AGGREGATE_HAVING`
+* `ORDER_BY_COLUMN`
+* `LIMIT`
+* `LIMIT_WITH_OFFSET`
```

Comment:
I thought we also had a lot of supported scalar functions

Reply:
You are right! Added issue ticket #84.